pkg/oauthserver/help_test.go

```diff
@@ -0,0 +1,97 @@
+package oauthserver_test
```

Comment:
the file name is confusing, `help_test.go` suggest you're testing something around the "help" topic in the oauthserver package.

Reply:
i removed all helper files

Reply:
done